presentation/src/main/java/com/anytypeio/anytype/presentation/multiplayer/ShareSpaceViewModel.kt

```diff
@@ -137,6 +137,13 @@ class ShareSpaceViewModel(
                     )
                 }
 
+                Timber.d("SpaceView: $spaceView, SpaceMembers: $spaceMembers, isCurrentUserOwner: $isCurrentUserOwner")
+
+                showIncentive.value = spaceView?.getIncentiveState(
```

Comment:
consider making incentive state not a side effect of this screen but a part of its ui state

Reply:
done